src/components/NewMessageInput.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import { Button, FormControl, InputGroup, Spinner } from "react-bootstrap";
+
+interface Props {
+  isLoading: boolean;
```

Comment:
`isLoading` should be managed by the send message function. Currently looks like `handleSendMessage` is inside this component, but `isLoading` outside.

Reply:
Done!